tests/models/git/test_modeling_git.py

```diff
@@ -383,11 +383,22 @@ class GitModelTest(ModelTesterMixin, GenerationTesterMixin, PipelineTesterMixin,
     all_model_classes = (GitModel, GitForCausalLM) if is_torch_available() else ()
     all_generative_model_classes = (GitForCausalLM,) if is_torch_available() else ()
     pipeline_model_mapping = (
-        {"feature-extraction": GitModel, "text-generation": GitForCausalLM} if is_torch_available() else {}
+        {"feature-extraction": GitModel, "image-to-text": GitForCausalLM, "text-generation": GitForCausalLM}
+        if is_torch_available()
```

Comment:
Keep `"image-to-text": GitForCausalLM` here (computed in an automatic way), but skip below.
In the end, we might decide `GitForCausalLM` is not for `image-to-text` task, and we will have to remove.